go.mod

```diff
@@ -2,11 +2,12 @@ module github.com/google/go-github/v21
 
 require (
 	github.com/golang/protobuf v1.2.0 // indirect
-	github.com/google/go-github v17.0.0+incompatible // indirect
+	github.com/google/go-github v17.0.0+incompatible
```

Comment:
Please revert your changes to this file.

Reply:
This is updated by `go mod tidy`, do I leave it as it was?

Reply:
Yes, please.  I'd like to only update `go.mod` and `go.sum` when upgrading this package version.